frontend/apps/reader/modules/readertoc.lua

```diff
@@ -137,7 +137,7 @@ end
 function ReaderToc:fillToc()
     if self.toc then return end
     if self.ui.document:canHaveAlternativeToc() then
-        if self.ui.doc_settings:readSetting("alternative_toc") then
+        if self.ui.doc_settings:has("alternative_toc") then
```

Comment:
You had to check (and so I had to) that I indeed did `self.ui.doc_settings:delSetting("alternative_toc")` when toggling it - before doing this change.
Isn't it safer sometimes to just trust readSetting, or use isTrue (which would work whether I have used delSetting or saveSetting(false)) ?

Reply:
I did switch quite a few things to isTrue to make those kinds of distinctions much clearer ;).

(e.g. limit readSetting to knobs that read a non-boolean value).

That one was aided by the following comment, but I did double-check ;).